composer.json

```diff
@@ -25,11 +25,11 @@
         "symfony/framework-bundle": "^3.4 || ^4.3 || ^5.0"
     },
     "require-dev": {
-        "matthiasnoback/symfony-dependency-injection-test": "^3",
+        "matthiasnoback/symfony-dependency-injection-test": "^4",
         "symfony/web-profiler-bundle": "^3.4 || ^4.3",
         "symfony/console": "^3.4 || ^4.3",
-        "phpunit/phpunit": "^7.0",
-        "symfony/phpunit-bridge": "^4.2",
+        "phpunit/phpunit": "^8.0",
+        "symfony/phpunit-bridge": "^5.2",
```

Comment:
The bridge can be updated with disregard with Symfony versions, it doesn't have any direct requirement, so it's better to have the most updated and fixed version.